packages/documentsearch-extension/schema/notebookShellWidgetListener.json

```diff
@@ -6,7 +6,7 @@
       "command": "documentsearch:start",
       "keys": ["Accel F"],
       "selector": ".jp-mod-searchable",
-      "disabled": true
+      "disabled": false
```

Comment:
Since these settings only include the keyboard shortcut override, maybe we could actually remove the whole file?